static/webpack-config.js

```diff
@@ -6,6 +6,7 @@ const HtmlPlugin = require('html-webpack-plugin');
 const RemovePlugin = require('remove-files-webpack-plugin');
 const { merge } = require('webpack-merge');
 const { parse } = require('comment-json');
+const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
```

Comment:
this is probably not going to change much, but if you move the require into the if statement does the build speed for regular builds change at all?

Reply:
that probably would help out a little

Reply:
ok I'm down to do that then!